wandb/sdk_py27/internal/sender.py

```diff
@@ -603,6 +608,9 @@ def _save_history(self, history_dict):
     def send_history(self, data):
         history = data.history
         history_dict = proto_util.dict_from_proto_list(history.item)
+        if self._settings._sync_tensorboard:
```

Comment:
I think this breaks grpc_server and `wandb sync`.
Until work is done cleaning up how settings is passed to the internal routines, we have a static dict created in both of those cases,  wandb/server, and wandb/sync

Running a regression test should catch both of these.